assets/src/edit-story/app/index.js

```diff
@@ -20,7 +20,10 @@
 import { ThemeProvider, StyleSheetManager } from 'styled-components';
 import stylisRTLPlugin from 'stylis-plugin-rtl';
 import PropTypes from 'prop-types';
-import KeyboardOnlyOutlines from '@moxy/react-keyboard-only-outlines';
+/**
+ * Internal dependencies
+ */
+import KeyboardOnlyOutlines from '../utils/keyboardOnlyOutline';
```

Comment:
Do we still want the `@moxy/react-keyboard-only-outlines` package in the mix? Aren't they overlapping in responsibilities?

Reply:
We don't, this PR removes it